### ISSUE TITLE
Unable to "deselect" an insurance policy from the dispensing window

### ISSUE THREAD

**Issue by wlthomson** [openmsupply/mobile#2530]
## Describe the bug

After selecting an insurance policy or creating a new one. The user cannot revert back to not using any insurance policy.

### To reproduce

1. Make a new prescription for a patient with an insurance policy.
2. Add an item and click next.
3. Select an insurance policy. 
4. Try to deselect the policy.
5. See error.

### Expected behaviour

I think the user should have a way of reverting back to "no selected policy"?

### Proposed Solution

Currently, there is a placeholder value `"Select an insurance policy"`. Could make this selectable? Alternatively, could add a small circular button next to the edit and add icons which reverts back to no policy (maybe an "undo" icon?).

### Version and device info

- App version: v4.0.1
- Tablet model: Android emulator.
- OS version: API 28.

### Additional context

N/A.

**Comment by joshxg** [comment]
Yeah I think a nice way is to add a little "cancel" X or something button next to the drop down. Although definitely normal priority imo, I reckon there is plenty of things to be done in 5.0.0 already!
